packages/pg-protocol/src/serializer.ts

```diff
@@ -160,7 +160,15 @@ const bind = (config: BindOpts = {}): Buffer => {
   writer.add(paramWriter.flush())
 
   // format code
-  writer.addInt16(binary ? ParamType.BINARY : ParamType.STRING)
+  if (binary) {
+    const columnFormatCount = 1
+    writer.addInt16(columnFormatCount)
+    writer.addInt16(ParamType.BINARY)
+  } else {
+    const columnFormatCount = 0
+    writer.addInt16(columnFormatCount)
+  }
+
```

Comment:
this has been fixed by https://github.com/brianc/node-postgres/pull/3494

Reply:
Great, thanks!